packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -150,21 +150,21 @@ curl -s -H "Authorization: SSWS ${api_token}" https://${yourOktaDomain}/api/v1/m
 
 <ApiOperation method="POST" url="/api/v1/meta/types/user/" />
 
-Creates a new User Type. A `default` User Type is automatically created along with your org, and you may add another 9 User Types for a maximum of 10.
+Creates a new User Type. A `default` User Type is automatically created with your org, and you can add another nine User Types for a maximum of 10.
 
-Okta periodically updates the default schema template used for new orgs, and new User Types will be based on the most up to date template. This means the properties with which a new User Type is initialized are not necessarily the same properties that your default type received.
+Okta periodically updates the default schema template used for new orgs. New User Types are based on the most up-to-date template. This means the properties that a new User Type is initialized with are not necessarily the same properties that your default type have received.
 
->**Note:** If you have modified your default schema, those changes will not propagate into this new User Type.
+>**Note:** If you have modified your default schema, those changes are not going to propagate into this new User Type.
```

Comment:
If you modified your default schema, those changes won't propagate into this new User Type.